package-lock.json

```diff
@@ -15317,7 +15317,7 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
Can you please remove these `"resolved": false` changes? I think they just get added when we run npm install/npm ci

Reply:
This has been resolved